snippets/language-ruby.cson

```diff
@@ -138,8 +138,11 @@
   'directory()':
     'prefix': 'dir'
     'body': 'File.dirname(__FILE__)'
-  'Insert do |variable| … end':
+  'Insert do … end':
     'prefix': 'do'
+    'body': 'do\n\t$0\nend'
+  'Insert do |variable| … end':
+    'prefix': 'dop'
     'body': 'do${1: |${2:variable}|}\n\t$0\nend'
```

Comment:
I don't think we need the first step for the `dop` snippet anymore, as it was to enable you to delete the `|variable|` in it's entirety. Instead I propose this:
```
'body': 'do |${1:variable}|\n\t$0\nend'
```

Otherwise, very nice addition! :+1: